pkg/internal/infrastructure/templates/main.tpl.tf

```diff
@@ -89,6 +93,8 @@ resource "azurerm_subnet" "{{ $workers }}" {
 {{- end }}
   address_prefixes          = ["{{ $subnet.cidr }}"]
   service_endpoints         = [{{ range $index, $serviceEndpoint := $subnet.serviceEndpoints }}{{ if $index }},{{ end }}"{{$serviceEndpoint}}"{{end}}]
+  private_link_service_network_policies_enabled = false
+  private_endpoint_network_policies_enabled = false
```

Comment:
Was the default behaviour for these 2 fields also changed?

Reply:
Ah, I was careless. Not this is a WIP commit from a different change I was preparing. Thank you for noticing